src/style.css

```diff
@@ -1,3 +1,91 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&displ0ay=swap';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+}
+
+.link:not(:first-child) {
+  margin-left: 20px;
+}
+
+.list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.element {
+  color: #000;
+  text-transform: uppercase;
+  text-align: center;
+  align-items: center;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  display: flex;
+}
+
+.element::after {
+  content: '';
+  position: absolute;
+  background-color: #00acdc;
+  border-radius: 8px;
+  height: 4px;
+  width: 0;
+  left: 0;
+  bottom: 0;
+}
+
+.element:active {
+  color: #00acdc;
+  box-shadow: 0 4px 0 #00acdc;
```

Comment:
When I click on menu item, this style adds the blue line under it. This is not needed.